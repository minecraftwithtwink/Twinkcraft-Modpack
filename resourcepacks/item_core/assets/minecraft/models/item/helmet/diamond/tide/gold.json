{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/helmet/base/diamond/tide",
        "layer1": "minecraft:trims/items/helmet_trim/tide_gold"
    }
}