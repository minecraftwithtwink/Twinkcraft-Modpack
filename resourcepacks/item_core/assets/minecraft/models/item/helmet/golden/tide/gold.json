{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/helmet/base/golden/tide",
        "layer1": "minecraft:trims/items/helmet_trim/tide_gold_darker"
    }
}